oxidation/client/src/components/CreateOrganisationModal.vue

```diff
@@ -0,0 +1,230 @@
+        <ion-item class="ion-margin-bottom">
+          <ion-label position="stacked">
+            {{ $t('CreateOrganization.deviceNameInputLabel') }}
+          </ion-label>
+          <ion-input
+            v-model="deviceName"
+            type="text"
+            :placeholder="$t('CreateOrganization.deviceNamePlaceholder')"
+          />
+        </ion-item>
+        <ion-item class="ion-margin-bottom">
+          <ion-label position="stacked">
+            {{ $t('CreateOrganization.password') }}
+          </ion-label>
+          <ion-input type="password" />
+        </ion-item>
+        <ion-item class="ion-margin-bottom">
+          <ion-label position="stacked">
+            {{ $t('CreateOrganization.confirmPassword') }}
+          </ion-label>
+          <ion-input type="password" />
+        </ion-item>
+      </div>
+    </ion-content>
+    <ion-footer>
+      <ion-toolbar>
+        <ion-buttons
+          v-if="pageStep === 1"
+          slot="primary"
+        >
+          <ion-button
+            @click="nextStep()"
+            :disabled="!firstPageIsFilled()"
+          >
+            {{ $t('CreateOrganization.next') }}
+          </ion-button>
+        </ion-buttons>
+        <ion-buttons
+          v-else
+          slot="primary"
+        >
+          <ion-button
+            @click="previousStep()"
+            slot="start"
+          >
+            {{ $t('CreateOrganization.previous') }}
+          </ion-button>
+          <ion-button type="submit">
+            {{ $t('CreateOrganization.done') }}
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-footer>
+  </ion-page>
+</template>
+
+<script setup lang = "ts" >
+import {
+  IonTitle,
+  IonToolbar,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonRadio,
+  IonRadioGroup,
+  IonCheckbox,
+  IonPage,
+  IonInput,
+  IonHeader,
+  IonContent,
+  IonButton,
+  IonButtons,
+  IonFooter,
+  IonIcon,
+  modalController
+} from '@ionic/vue';
+
+import {
+  close
+} from 'ionicons/icons';
+import { ref } from 'vue';
+
+const showTos = ref('parsecServer');
+const ownServerUrl = ref('');
+const acceptTos = ref(false);
+const pageStep = ref(1);
+const email = ref('');
+const fullname = ref('');
+const organisation = ref('');
+const deviceName = ref('');
+
+function nextStep(): void {
+  pageStep.value = 2;
+}
+
+function previousStep(): void {
+  pageStep.value = 1;
+}
+
+/* temporary test, we will do a more complete form validation */
+function firstPageIsFilled(): boolean {
+  return email.value.length > 0 && fullname.value.length > 0 && organisation.value.length > 0
+  && ( acceptTos.value || ownServerUrl.value.length > 0 );
+}
+
+function closeModal(): Promise<boolean> {
+  return modalController.dismiss(null, 'cancel');
+}
+
+</script>
+
+<style lang="scss" scoped>
+.flex-row {
+  @media screen and (min-width: 576px) {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+
+    ion-item {
+      width: 48%;
+
```

Comment:
Maybe you need a linter ?

Reply:
Well for now we have this ^^ https://github.com/Scille/parsec-cloud/blob/05db91ce699801e3071d0978bef6c48619831062/oxidation/client/.eslintrc.js#L26